aws-datastore/src/test/resources/update-blog-owner-with-predicate.txt

```diff
@@ -0,0 +1,13 @@
+mutation UpdateBlogOwner($input:UpdateBlogOwnerInput!, $condition:ModelBlogOwnerConditionInput) {
+  updateBlogOwner(input:$input, condition:$condition) {
+    blog {
+      id
+    }
+    id
+    name
+    wea
```

Comment:
wea?

Reply:
Heh. So, I grabbed this schema from a developer. Francisco, I think?  He was actively testing something. Anyway, our BlogOwner has a `wea` field baked into it. I don't recall where I got the schema, or what was being tested with the `wea`.

Reply:
In the jungle, the mighty jungle.. o-wimowea 
, o-wimowea.